training/src/app/chuck-noris/chuck-noris-main/chuck-norris-main.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component } from '@angular/core';
+import { JokeGeneratorService } from "../joke-generator.service";
+import { Observable, Subject } from "rxjs";
+import { Joke } from "../joke";
+
+@Component({
+	selector: 'app-chuck-norris-main',
+	templateUrl: './chuck-norris-main.component.html',
+	styleUrls: ['./chuck-norris-main.component.scss']
+})
+
+export class ChuckNorrisMain {
+	public jokeList$: Observable<Joke[]>;
+	public joke$: Observable<Joke>;
+	public clicked: string              = 'random';
```

Comment:
doesnt sounds like a good name, how come its "clicked" and then the value is a string of 'random'?
please find a better name